integration_test/test/recording_endpoint_test.exs

```diff
@@ -162,7 +162,9 @@ defmodule Membrane.RTC.RecordingEndpointTest do
     setup_mock_http_request()
 
     recording_endpoint =
-      create_recording_endpoint(rtc_engine, [{Storage.S3, %{credentials: @credentials}}])
+      create_recording_endpoint(rtc_engine, [
+        {Storage.S3, %{credentials: @credentials, path_prefix: ""}}
```

Comment:
I think that if `path_prefix` is not provided a default value should be used which would be `""` .